build/cygnus.js

```diff
@@ -1,29 +1,28 @@
-var _this2 = this,
-    _arguments = arguments;
-
 /* global cygnus, Blob, Worker, XMLHttpRequest */
 
+const _arguments = arguments;
+
 module.exports = {
   supportsHistory: !!window.history,
   supportsWorkers: !!window.Worker,
   supportsPromises: !!Promise,
   ready: false,
   pages: {},
-  init: () => {
+  init: function () {
```

Comment:
Arrow functions do not create their own scope, but the `function` keyword does. Therefore you can use `this` within one of these functions. My first pass was kind of a straight conversion and I wasn't paying attention to scope, but all the methods should use this type of function for `this` access.